src/Root.js

```diff
@@ -18,20 +18,38 @@ console.disableYellowBox = true; // eslint-disable-line no-console
 // }
 // /* ------------------------ */
 
-const Root = () => (
-  <ErrorBoundary>
-    <ReduxProvider store={store}>
-      <VertxProvider>
-        <ThemeProvider>
-          <LayoutProvider>
-            <GoogleProvider>
-              <App />
-            </GoogleProvider>
-          </LayoutProvider>
-        </ThemeProvider>
-      </VertxProvider>
-    </ReduxProvider>
-  </ErrorBoundary>
-);
+class Root extends Component {
+  componentDidMount() {
+    this.attachStoreToWindow();
```

Comment:
Does this need to be in `componentDidMount()`? I have successfully done this before simply under the imports and have not run into any issues, curious to hear if you ran into any issues / if you tried this?